### ISSUE TITLE
Expunge Assist: Update alt text for Expunge Assist image to adhere to WCAG

### ISSUE THREAD

**Issue by plang-psm** [hackforla/website#3998]
### Prerequisite
1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started page](https://www.hackforla.org/getting-started).
2. Please make sure you have read our **[Hack for LA Contributing Guide](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md)** before you claim/start working on an issue.

### Overview
As a developer, we need to provide clear and descriptive alt text on the `Expunge Assist` page so that we adhere to the Web Content Accessibility Guidelines (WCAG) and we can achieve our mission of inclusivity.

### Action Items
- [x] Change the image alt property value within `_projects/expunge-assist.md`:

**From:**
```
Image of a criminal record
```

**To:**
```
Expunge Assist
```

**For PM, Merge Team, or Tech Lead**
- [x] Once the pull request associated with this issue is approved and merged, please update and edit epic #3866 by
  - [x] Checking off the dependency for this issue
  - [x] If all dependencies are checked off, please move issue to the New Issue Approval column and remove the Dependency label

### Resources/Instructions
- https://www.hackforla.org/projects/expunge-assist

#### File and Code links you will need to work on this issue
- Directory to find the page in once you have it in your IDE: `_projects/expunge-assist.md`
- Click on the filename below to see the source file code:
https://github.com/hackforla/website/blob/gh-pages/_projects/expunge-assist.md
- This issue is part of epic #3866 

#### Background information about the Jekyll framework (optional reading)
- [Jekyll Data Files](https://jekyllrb.com/docs/datafiles/)
- [Jekyll Front Matter](https://jekyllrb.com/docs/front-matter/)

#### Information about WCAG and alt text (optional reading)
- [WCAG](https://www.w3.org/WAI/standards-guidelines/wcag/)
- [Web Accessibility Tutorials: Decorative Images](https://www.w3.org/WAI/tutorials/images/decorative/)
- [WCAG section on alt text](https://www.w3.org/WAI/WCAG21/Techniques/html/H37.html)
- [Alt text decision Tree ](https://www.w3.org/WAI/tutorials/images/decision-tree/)
- [FAE rule 4: alt text should be less than 100 characters](https://fae.disability.illinois.edu/rulesets/IMAGE_4_EN/)

**Comment by arora-aryan** [comment]
Availability: 2.5 hours/week 
ETA: 2 weeks

**Comment by jdingeman** [comment]
Hi @arora-aryan, I saw your ETA was going to be 2 weeks from your update. Just wanted to check in with you if you have any updates.

Thanks!

**Comment by jdingeman** [comment]
Hi @arora-aryan - since we have not heard any updates, we are going to unassign you from this issue and place it back into the prioritized backlog.

**Comment by mademarc** [comment]
Availability: 4/3/2023
ETA: 4/6/2023

**Comment by qngyn** [comment]
Availability: 4/1/2023
ETA: 4/1/2023

**Comment by jdingeman** [comment]
@qngyn - are you still working on this issue?

**Comment by ExperimentsInHonesty** [comment]
@qngyn I am removing you from this issue, since you have not provided an update in 3 week and you said you would be done on April 1st.
